content/docs/typechecking-with-proptypes.md

```diff
@@ -119,9 +119,9 @@ MyComponent.propTypes = {
 };
 ```
 
-### Requiring Single Child {#requiring-single-child}
+### Membutuhkan Satu Komponen Anak {#requiring-single-child}
 
-With `PropTypes.element` you can specify that only a single child can be passed to a component as children.
+Dengan menggunakan `PropTypes.element` kita dapat menentukan hanya menerima satu komponen anak yang dapat dikirimkan ke komponen lain.
```

Comment:
You -> kamu atau ada preferensi kenapa pakai kita?

Reply:
kita dapat menentukan hanya menerima satu komponen anak yang dapat dikirimkan ke komponen lain. -> kamu dapat menentukan bahwa hanya satu komponen anak yang dapat dikirimkan ke komponen lain.

Gimana om @rizafahmi ?

Reply:
Kata-kata kita hanya preferensi pribadi. Kalau disini menggunakan kamu, saya ngikut aja. Maaf newbie 🙇‍♂️

Reply:
Di issue lain sempat lihat sekilas pakai Anda